ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -402,6 +386,51 @@ Item {
         }
     }
 
+    Loader {
+        id: createChatOrCommunity
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: Style.current.padding
+        active: communityData.amISectionAdmin && communityData.canManageUsers
```

Comment:
the `enabled` property of the menu item seems to indicate that u just need `amISectionAdmin`.
If I have `canManageUsers` only then there will be not menu item?
Seems odd?